### ISSUE TITLE
Aura and Synapse will work then stop working after a few seconds

### ISSUE THREAD

**Issue by weavedawg74** [AndrewBabbitt97/AuraConnect#18]
I had been able to get this work previously, but yesterday I noticed that the Synapse effect on the Aura module is showing as connected and the effects are rendering inside Synapse. However the effects only work for about 5 seconds then just get stuck on a static color.

**Comment by tark95** [comment]
Hm, odd. This only happens to me when I try to mess with the lighting effects via Aura. I will set something like a rainbow effect in Armoury Crate and after a few seconds it will switch to the profile I have activated in Synapse since it prioritizes Synapse (as it should).

Have you tried to reinstall everything?

**Comment by AndrewBabbitt97** [comment]
Logs can be very helpful here (C:\ProgramData\AuraConnect) , I may add more expansive logging to look at situations like this.

**Comment by weavedawg74** [comment]
Here's the log from the past 2 days:

2020-03-03T13:34:36.7722897-06:00	INFO	[Microsoft.Hosting.Lifetime]	[0]	Application started. Hosting environment: Production; Content root path: C:\Program Files (x86)\ASUS\AURACONNECT\
2020-03-03T13:34:37.7928843-06:00	INFO	[AuraConnect.Worker]	[0]	Initializing Aura Connect...
2020-03-03T13:35:23.9125670-06:00	INFO	[AuraConnect.Worker]	[0]	Found Device: AURA - Corsair Vengeance RGB Pro - 40 Lights
2020-03-03T13:35:23.9127306-06:00	INFO	[AuraConnect.Worker]	[0]	Found Device: AURA - Desktop - 8 Lights
2020-03-03T13:35:23.9127535-06:00	INFO	[AuraConnect.Worker]	[0]	Found Device: AURA - Mainboard_Master - 5 Lights
2020-03-03T13:35:23.9127622-06:00	INFO	[AuraConnect.Worker]	[0]	Aura Connect started successfully!
2020-03-03T13:35:24.4447480-06:00	INFO	[AuraConnect.Worker]	[0]	Razer Chroma Broadcast API connected
2020-03-03T13:57:56.9103334-06:00	INFO	[Microsoft.Hosting.Lifetime]	[0]	Application started. Hosting environment: Production; Content root path: C:\Program Files (x86)\ASUS\AURACONNECT\
2020-03-03T13:57:57.9260371-06:00	INFO	[AuraConnect.Worker]	[0]	Initializing Aura Connect...
2020-03-03T13:58:41.2130929-06:00	INFO	[AuraConnect.Worker]	[0]	Found Device: AURA - Corsair Vengeance RGB Pro - 40 Lights
2020-03-03T13:58:41.2132450-06:00	INFO	[AuraConnect.Worker]	[0]	Found Device: AURA - Desktop - 8 Lights
2020-03-03T13:58:41.2133165-06:00	INFO	[AuraConnect.Worker]	[0]	Found Device: AURA - Mainboard_Master - 5 Lights
2020-03-03T13:58:41.2133364-06:00	INFO	[AuraConnect.Worker]	[0]	Aura Connect started successfully!
2020-03-03T13:58:42.2065895-06:00	INFO	[AuraConnect.Worker]	[0]	Razer Chroma Broadcast API disconnected
2020-03-03T13:58:48.2278549-06:00	INFO	[AuraConnect.Worker]	[0]	Razer Chroma Broadcast API connected
2020-03-03T14:20:00.0057176-06:00	INFO	[Microsoft.Hosting.Lifetime]	[0]	Application started. Hosting environment: Production; Content root path: C:\Program Files (x86)\ASUS\AURACONNECT\
2020-03-03T14:20:01.0044421-06:00	INFO	[AuraConnect.Worker]	[0]	Initializing Aura Connect...
2020-03-03T14:20:40.3980916-06:00	INFO	[AuraConnect.Worker]	[0]	Found Device: AURA - Corsair Vengeance RGB Pro - 40 Lights
2020-03-03T14:20:40.3982471-06:00	INFO	[AuraConnect.Worker]	[0]	Found Device: AURA - Desktop - 8 Lights
2020-03-03T14:20:40.3982680-06:00	INFO	[AuraConnect.Worker]	[0]	Found Device: AURA - Mainboard_Master - 5 Lights
2020-03-03T14:20:40.3982823-06:00	INFO	[AuraConnect.Worker]	[0]	Aura Connect started successfully!
2020-03-03T14:20:40.4409284-06:00	INFO	[AuraConnect.Worker]	[0]	Razer Chroma Broadcast API connected
2020-03-04T07:07:27.0394734-06:00	INFO	[AuraConnect.Worker]	[0]	Razer Chroma Broadcast API disconnected
2020-03-04T07:07:31.0070342-06:00	INFO	[AuraConnect.Worker]	[0]	Razer Chroma Broadcast API connected
2020-03-04T07:21:20.7922122-06:00	INFO	[Microsoft.Hosting.Lifetime]	[0]	Application is shutting down...
2020-03-04T07:21:29.5525604-06:00	INFO	[Microsoft.Hosting.Lifetime]	[0]	Application started. Hosting environment: Production; Content root path: C:\Program Files (x86)\ASUS\AURACONNECT\
2020-03-04T07:21:30.5511323-06:00	INFO	[AuraConnect.Worker]	[0]	Initializing Aura Connect...
2020-03-04T07:22:09.9547215-06:00	INFO	[AuraConnect.Worker]	[0]	Found Device: AURA - Corsair Vengeance RGB Pro - 40 Lights
2020-03-04T07:22:09.9548879-06:00	INFO	[AuraConnect.Worker]	[0]	Found Device: AURA - Desktop - 8 Lights
2020-03-04T07:22:09.9549075-06:00	INFO	[AuraConnect.Worker]	[0]	Found Device: AURA - Mainboard_Master - 5 Lights
2020-03-04T07:22:09.9549166-06:00	INFO	[AuraConnect.Worker]	[0]	Aura Connect started successfully!
2020-03-04T07:22:09.9996563-06:00	INFO	[AuraConnect.Worker]	[0]	Razer Chroma Broadcast API connected
2020-03-04T07:31:59.7772826-06:00	INFO	[Microsoft.Hosting.Lifetime]	[0]	Application is shutting down...
2020-03-04T07:32:02.0911588-06:00	INFO	[Microsoft.Hosting.Lifetime]	[0]	Application started. Hosting environment: Production; Content root path: C:\Program Files (x86)\ASUS\AURACONNECT\
2020-03-04T07:32:03.0877075-06:00	INFO	[AuraConnect.Worker]	[0]	Initializing Aura Connect...
2020-03-04T07:32:42.2001596-06:00	INFO	[AuraConnect.Worker]	[0]	Found Device: AURA - Corsair Vengeance RGB Pro - 40 Lights
2020-03-04T07:32:42.2002900-06:00	INFO	[AuraConnect.Worker]	[0]	Found Device: AURA - Desktop - 8 Lights
2020-03-04T07:32:42.2003120-06:00	INFO	[AuraConnect.Worker]	[0]	Found Device: AURA - Mainboard_Master - 5 Lights
2020-03-04T07:32:42.2003196-06:00	INFO	[AuraConnect.Worker]	[0]	Aura Connect started successfully!
2020-03-04T07:32:42.2386286-06:00	INFO	[AuraConnect.Worker]	[0]	Razer Chroma Broadcast API connected

**Comment by weavedawg74** [comment]
So I just watched a video from Techblock about your software and he pointed out something that may make sense for my issue. He said that he encountered some issues with trying to do a rainbow wave on everything that would cause Aura devices to kind of lag when changing colors. The profile I'm trying to use is a a static teal color with a layer of transparent and dark teal that goes really quickly, giving it a shimmering water effect, and then I also have a red ripple that occurs on key press. Sounds weird but it's all a part of my Shark Attack theme. 

I'm currently at work so I can't test it, but I think that I may need to disable the wave that I have on the Aura module and perhaps that will help fix the issue. Because what happens for me is that the static color works, the wave will work momentarily then it will freeze. I know it's a freeze because there are times where the red ripple will be active and then freeze on the mobo ROG logo. So I'm thinking that perhaps it's too much going on at once for the software and it causes it to freeze.

A couple of other questions:

Are there any specific settings that I should enable on the Aura side of things? 

I've made sure that my Armory Crate is up-to-date and it installs Aura Creator, is that something that could cause these issues?
